wiki/en/en-Contribution.md

```diff
@@ -9,19 +9,19 @@ permalink: "/wiki/Contribution"
 
 ## Jamulus Software
 
-If you want to contribute code or have a feature request, [open an issue on GitHub](https://github.com/corrados/jamulus/issues/) (preferred) or [post your idea on the developer forum](https://sourceforge.net/p/llcon/discussion/developerforum/) if you don't have a GitHub account. This allows us to discuss the changes before e.g. making a pull request (be sure to read main dev's [Volker's manifesto for contributions on GitHub](https://github.com/corrados/jamulus/issues/596) and the [CONTRIBUTING file](https://github.com/corrados/jamulus/blob/master/CONTRIBUTING.md))
+If you want to contribute code or have a feature request, [open an issue on GitHub](https://github.com/corrados/jamulus/issues/) (preferred) or [post your idea on the developer forum](https://sourceforge.net/p/llcon/discussion/developerforum/) if you don't have a GitHub account. This allows us to discuss the changes before e.g. making a pull request (be sure to read main dev's [Volker's manifesto for contributions on GitHub](https://github.com/jamulussoftware/jamulus/issues/596) and the [CONTRIBUTING file](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md))
 
-See the [Jamulus GitHub repo](https://github.com/corrados/jamulus) for further information.
+See the [Jamulus GitHub repo](https://github.com/jamulussoftware/jamulus) for further information.
 
 ## Jamulus Website
 
-* If you find a mistake, typo or something out of date (in any language) on the website, you can [open an issue on GitHub](https://github.com/jamulussoftware/jamuluswebsite/issues) or, if you don't have a GitHub account on [the relevant discussion forum](https://sourceforge.net/p/llcon/discussion/) on SourceForge.
+* If you find a mistake, typo or something out of date (in any language) on the website, you can [open an issue on GitHub](https://github.com/jamulussoftware/jamuluswebsite/issues) or if you don't have a GitHub account, on [the relevant discussion forum](https://sourceforge.net/p/llcon/discussion/) on SourceForge.
```

Comment:
There should be a comma before and after `if you don't have a GitHub account`.  Can't remember what part of speech that makes it...

Reply:
Done.